### PR TITLE
Models

### PR DESCRIPTION
**Pull request by AdleenaLobo**
Did changes to the search event by name to be case insenstive

### CODE REVIEW

File: controllers/eventController.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import Organizer from "../models/organiser.js";
-import Event from "../models/event.js";
+import Event from "../models/event.
```
Comment (CastelShal) [inline controllers/eventController.js]: Uhhhhh, why remove the .js?

File: controllers/eventController.js [hunk 2]
```diff
@@ -43,7 +45,12 @@ export async function getEventByName(req, res) {
   try {
     const event = await Event.findAll({
       where: {
-        eventName: req.params.eventName,
+        [Op.and]: [
+          sequelize.where(
+            sequelize.fn("lower", sequelize.col("eventName")),
+            req.params.eventName.toLowerCase()
```
Comment (CastelShal) [inline controllers/eventController.js]: Just a minor change, as shown in the code here

https://stackoverflow.com/questions/41728023/sequelize-case-insensitive-like#answer-41732931

**Review by CastelShal — CHANGES_REQUESTED** [review]
Just add the one line and add the .js back
(⁠◠⁠‿⁠◕⁠)

**Comment by AdleenaLobo** [comment]
made changes shalome

**Comment by CastelShal** [comment]
> made changes shalome
> 

Huhhhhhhhh

**Review by CastelShal — APPROVED** [review]

**Review by CastelShal — APPROVED** [review]

**Pull request merged**